layers/+spacemacs/spacemacs-org/packages.el

```diff
@@ -59,6 +59,11 @@
                     (1 font-lock-comment-face prepend)
                     (2 font-lock-function-name-face)
                     (3 font-lock-comment-face prepend))))
+      ;; changing themes can leave artifacts in org mode, it should reload
+      (add-hook 'spacemacs-post-theme-change-hook
+                '(lambda ()
+                   (when (featurep 'org)
+                     (org-save-outline-visibility 'use-markers (org-mode-restart)))))
```

Comment:
Would be better to extract that to a function and then add it to hook. Almost all the functions added to hooks in spacemacs are named functions (maybe "Unwritten Rule") which helps if someone wants to remove it from the hook.